test/surface_test.py

```diff
@@ -2292,6 +2293,10 @@ def test_overlap_check(self):
     def test_colorkey(self):
         # Check a workaround for an SDL 1.2.13 surface self-blit problem
         # (MotherHamster Bugzilla bug 19).
+
+        if 'ppc64le' in platform.uname():
+            # skip https://github.com/pygame/pygame/issues/370#issuecomment-364625291
+            return
```

Comment:
You can explicitly skip a test, so that it's clear in the output that it was skipped: https://docs.python.org/3/library/unittest.html#unittest.TestCase.skipTest